crates/holochain/examples/websocket_stress.rs

```diff
@@ -0,0 +1,125 @@
+use holochain::{
+    conductor::{
+        api::{AdminRequest, AdminResponse},
+        Conductor,
+    },
+};
+use tempfile::TempDir;
+use holochain_conductor_api::conductor::ConductorConfig;
+use holochain_conductor_api::conductor::KeystoreConfig;
+use holochain_conductor_api::AdminInterfaceConfig;
+use holochain_conductor_api::InterfaceDriver;
+use tokio_tungstenite::*;
+use tungstenite::Message;
+use futures::sink::SinkExt;
+use futures::stream::StreamExt;
+use holochain_websocket::WireMessage;
+
+use std::sync::atomic::{AtomicU64, Ordering};
+
+static ID: AtomicU64 = AtomicU64::new(1);
+
+use holochain_serialized_bytes as hsb;
+
+const MINUTES_LONG_BEHAVED_COUNT: usize = 10;
+const MINUTES_LONG_BAD_COUNT: usize = 10;
+const SECONDS_LONG_BEHAVED_COUNT: usize = 100;
+const SECONDS_LONG_BAD_COUNT: usize = 100;
+
+static CONS_MADE: AtomicU64 = AtomicU64::new(0);
+static MSGS: AtomicU64 = AtomicU64::new(0);
+static GOOD_CLOSE: AtomicU64 = AtomicU64::new(0);
+static BAD_CLOSE: AtomicU64 = AtomicU64::new(0);
+
+#[tokio::main(flavor = "multi_thread")]
+pub async fn main() {
+    let tmp_dir = TempDir::new().unwrap();
+    let environment_path = tmp_dir.path().to_path_buf();
+    let config = ConductorConfig {
```

Comment:
@thedavidmeister where does a `ConductorConfig` fit in with your network toplogy model?